airflow-core/src/airflow/ui/public/i18n/locales/de/common.json

```
@@ -39,6 +39,7 @@
     "fileLocation": "Ablagepfad",
     "hasTaskConcurrencyLimits": "Hat der Task Limitierungen zur Parallelität",
     "lastExpired": "Letztmaliger Ablauf",
+    "lastParseDuration": "Letzte Dag Parse-Dauer",
```

```suggestion
    "lastParseDuration": "Letzte Dag Einlesedauer",
```

wenn lastParsed "Letztmalig Eingelesen" ist... ? :) Nur ein Vorschlag

---

Klingt plausibel. "Eingelesen ist natürlicher aber im Deutschen ist "parsen" ja auch ein normaler Term... ja ich folge mal dem Vorschlag,